invenio_rdm_records/services/config.py

```diff
@@ -230,6 +231,7 @@ class RDMRecordServiceConfig(RecordServiceConfig, ConfiguratorMixin):
     # Schemas
     schema = RDMRecordSchema
     schema_parent = RDMParentSchema
+    schema_parent_access_settings = AccessSettingsSchema
```

Comment:
```suggestion
    schema_access_settings = AccessSettingsSchema
```
? similar to grant and link?